### ISSUE TITLE
Some natvis functionallity not working for VSCode qt5.natvis

### ISSUE THREAD

**Issue by borro0** [microsoft/MIEngine#1101]
I'm no expert on the matter.
But I see that for example QVariant defined in qt5.natvis is not working for Linux gdb type program.
https://github.com/inviwo/inviwo/blob/master/tools/natvis/qt5.natvis

Could this be because natvis is not fully supported on gdb type programs?
And if so, could you please add support for that QVariant functionality?

**Comment by WardenGnaw** [comment]
Yes. the natvis reader for VS Code is not as extensive as the VS one.
